.github/workflows/release.yml

```diff
@@ -202,7 +208,7 @@ jobs:
           # to retry a release action execution that failed due to a transient issue.
           # In that case, we expect the commit to be releasable as-is, i.e. the changelog should have already
           # been processed.
-          git fetch --unshallow
+          git fetch
```

Comment:
FYI, specifying `--unshallow` in a repository with `fetch-depth: 0` will fail with the following error:
```
fatal: --unshallow on a complete repository does not make sense
```